### ISSUE TITLE
Keyboard lighting gets stuck

### ISSUE THREAD

**Issue by Paladinleeds** [roxaskeyheart/Chromatics#74]
After finishing ARF as SAM, I leave the duty and my keyboard doesn't realise I've left the duty and freezeframed on the end of the boss (so kenki guage doesn't reset, etc, even changed classes and it's still thinking I'm on SAM). Had to reload Chromatics to fix.

This seems to apply to all duties. I've also noticed after being in the Sirensong Sea.

EDIT: I should add that the keyboard and mouse I have are Razer (Cynosa and Naga Trinity respectively)

UPDATE: This isn't just happening as I'm leaving duty anymore. This is happening randomly and frequently. A fix seems to be to disable the keyboard then re-enable it. It happens regardless of which job/class I am on.

N.B.: I should add I haven't had chance to fully test the other Chromatics enabled devices I have (Naga Trinity and Firefly) but it seems to be affecting them as well.

**Comment by AleanaA** [comment]
Can confirm that it's happening on the Razer Ornata Chroma as well.

**Comment by nightcipher** [comment]
Unsure if this is a related issue, but when using Chromatics with the Blackwidow Chroma V2, the following occurs:

Upon trying to enter the Eureka Anemos duty, the keyboard will flash to signal that the duty is ready to commence, as it correctly does with other Duty Finder tasks (such as Expert Roulette, Trials Roulette, etc). However, upon entering the Eureka Anemos instance, they keyboard will continue to flash until the duty is left. 

EDIT: Confirmed that this is happening with other duties as well, not just Eureka. The Duty Bell feature seems to have something causing it to loop endlessly.

**Comment by Paladinleeds** [comment]
@nightcipher I was going to create a separate issue about this, but since it's been brought up here I'll add it here. With my Cynosa Chroma, when the duty finder bell goes off and someone declined, it keeps flashing red until the next DF bell pop. Like you said, something getting stuck.

**Comment by logicallysynced** [comment]
Is this still happening in 2.3.9a?

https://chromaticsffxiv.com/chromatics2/Chromatics2.3.9aBETA-Portable.zip
(Don't forget to unblock before extracting and run as administrator)

(Note Duty Finder Bell is currently not available)

**Comment by Paladinleeds** [comment]
Looks to have been resolved prior to the new version being released. Seems the 4.3 update that was done resolved it. Closing. Will re-open should the issue rear its head again.

**Comment by logicallysynced** [comment]
What's the exact reproduction steps?

**Comment by Paladinleeds** [comment]
I don't know how best to reproduce it. It seems to be random when it does it. It has however done it just now while I've been in Eureka.

**Comment by logicallysynced** [comment]
See if you can reproduce this issue in the 2.4.2a beta.

**Comment by logicallysynced** [comment]
Closing due to the impending release of Chromatics 3.x.

Chromatics 3 is a complete rebuild of Chromatics from the ground up, and as such most previous issues are likely now invalidated. Please try the new version when released and then report any new issues as they occur.
